src/CoreWCF.RabbitMQ/tests/IntegrationTests.cs

```diff
@@ -18,46 +21,198 @@ namespace CoreWCF.RabbitMQ.Tests
     public class IntegrationTests
     {
         private readonly ITestOutputHelper _output;
-        public const string QueueName = "wcfQueue";
 
         public IntegrationTests(ITestOutputHelper output)
         {
             _output = output;
         }
 
-        [Fact(Skip ="need rabbitmq")]
-        public async Task ReceiveMessage_ServiceCall_Success()
+        [Fact(Skip = "Requires RabbitMQ host with SSL")]
+        public void ClassicQueueWithTls_ReceiveMessage_Success()
         {
-            IWebHost host = ServiceHelper.CreateWebHostBuilder<Startup>(_output).Build();
+            IWebHost host = ServiceHelper.CreateWebHostBuilder<ClassicQueueWithTLSStartup>(_output, nameof(ClassicQueueWithTls_ReceiveMessage_Success)).Build();
             using (host)
```

Comment:
I didn't mean to add it at the call point take a look at the implementation [here](https://github.com/CoreWCF/CoreWCF/blob/main/src/CoreWCF.Http/tests/Helpers/ServiceHelper.cs#L88) where it uses `[CallerMemberName] string callerMethodName = ""` as a parameter in the CreateWebHostBuilder method. Let the compiler do the work for you.